src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1184,10 +1184,9 @@ public static string SensitivityParameterIsInvalid(string name)
       public static readonly string CovarianceMatrixCannotBeCalculated = "Covariance matrix cannot be calculated";
       public static readonly string CorrelationMatrixCannotBeCalculated = "Correlation matrix cannot be calculated";
 
-      public static string CannotSelectTheSamePartialProcessMoreThanOnce(string name)
-      {
-         return $"'{name}' cannot be selected more than once.";
-      }
+      public static string CannotSelectTheSamePartialProcessMoreThanOnce(string name) => $"'{name}' cannot be selected more than once.";
+
+      public static string CannotSelectTheObservedDataMoreThanOnce(string name) => $"'{name}' cannot be selected more than once for the same output.";
```

Comment:
We still have a restriction. Same output, same observed data. That does not make sense